Composer/packages/electron-server/src/electronWindow.ts

```diff
@@ -47,6 +47,7 @@ export default class ElectronWindow {
     this.currentBrowserWindow = new BrowserWindow(browserWindowOptions);
     this.currentBrowserWindow.on('page-title-updated', (ev) => ev.preventDefault()); // preserve explicit window title
     this.currentBrowserWindow.webContents.on('new-window', this.onOpenNewWindow.bind(this));
+    this.currentBrowserWindow.webContents.userAgent = 'Electron';
```

Comment:
Why are you changing the user agent string? The user agent string already contains electron along with a lot of other useful information. 

![image](https://user-images.githubusercontent.com/17039790/122798625-09610e80-d27e-11eb-98fd-3a6ed38955f2.png)

Reply:
I don't remember making this change, and you're right that it's confusing. I'll revert it.